flaml/automl/automl.py

```diff
@@ -1215,6 +1214,7 @@ def fit(
         free_mem_ratio=0,
         metric_constraints=None,
         custom_hp=None,
+        time_col=None,
```

Comment:
Does it still default to first column if no `time_col` is passed it? this is crucial to keep user's current code working with new version

Reply:
also, it would be great if a description is added to docstring

Reply:
Update: I see that it defaults to first col in flaml/automl/task/time_series_task.py

Reply:
Docstring description added